CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.2.1 (2018-01-15)
+
+### Fixes
+
+- Fix an issue that prevented fetching tags for private DockerHub repositories [weaveworks/flux#897](https://github.com/weaveworks/flux/pull/897)
```

Comment:
It may have caused problems for self-hosted registries too -- perhaps "private image repositories on DockerHub (and self-hosted registries)"

Reply:
Good call